### PR TITLE
Updated .NET Core Linux Prerequisites to remove specific versions

### LINKED ISSUE #4086: Update .NET Core Linux Prerequisites to latest
# Update .NET Core Linux Prerequisites to latest 

Remove installation instructions for each distro, and replace with links to .NET download page.

4. Suggest reviewers

    - If you know someone who can provide feedback, use '@' to ask them to review.

### PR DESCRIPTION
**Pull request by JRAlexander**
# Updated .NET Core Linux Prerequisites to remove specific versions

## Summary

Updated .NET Core Linux Prerequisites to remove specific versions of Linux distributions and link to their respective download instruction pages.

[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/core/linux-prerequisites?branch=pr-en-us-4099&tabs=netcore2x)

Fixes #4086

### CODE REVIEW

**Review by leecow — APPROVED** [review]
lgtm

File: docs/core/linux-prerequisites.md [hunk 1]
```diff
@@ -194,59 +157,13 @@ Enable .NET Core 1.0 SDK and Runtime for your environment:
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: this step 4 looks out of place

File: docs/core/linux-prerequisites.md [hunk 2]
```diff
@@ -194,59 +157,13 @@ Enable .NET Core 1.0 SDK and Runtime for your environment:
...
-   ```bash
-   sudo apt-get install dotnet-sdk-2.1.3
-   ```
-
-4. Run the `dotnet --version` command to prove the installation succeeded.
-
-   ```bash
-   dotnet --version
-   ```
+2. To ensure you have the latest installation information, use the [.NET Core 2.x SDK and Runtime Installer instructions](https://www.microsoft.com/net/learn/get-started/linuxubuntu) for supported Ubuntu and Linux Mint distributions\versions (64 bit).
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: mixing steps in and out of the tab makes it hard to follow. I almost thought it was a mistake to start at step 2 when I noticed you had step 1 before the tab.

File: docs/core/linux-prerequisites.md [hunk 3]
```diff
@@ -110,50 +110,13 @@ The installer bash script is used in automation scenarios and non-admin installa
...
-   ```bash
-   yum install rh-dotnet20
-   ```
-
-Enable .NET Core 2.0 SDK/Runtime for your environment:
-
-   ```bash
-   scl enable rh-dotnet20 bash
-   ```
+To ensure you have the latest installation information, use the [.NET Core 2.x SDK and Runtime Installer instructions](https://www.microsoft.com/net/learn/get-started/linuxredhat) for supported RHEL versions.
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: Do you really need to say "To ensure you have the latest installation information,"?

Use -> Follow

File: docs/core/linux-prerequisites.md [hunk 4]
```diff
@@ -383,9 +254,9 @@ To install .NET Core on Debian 8 or Debian 9 (64 bit):
 
 ---
 
-## Install .NET Core for Fedora 24, Fedora 25, or Fedora 26 (64 bit)
+## Install .NET Core for supported Fedora versions (64 bit)
 
-To install .NET Core 2.x on Fedora 26 or Fedora 25, or .NET Core 1.x on Fedora 24:
+To install .NET Core 2.x on supported Fedora versions, or .NET Core 1.x on Fedora 24:
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: To install .NET Core on supported Fedora versions?

File: docs/core/linux-prerequisites.md [hunk 5]
```diff
@@ -529,77 +350,19 @@ To install .NET Core for CentOS 7.1 (64 bit) & Oracle Linux 7.1 (64 bit):
...
-   ```
-
-5. Add dotnet to your PATH.
-
-   ```bash
-   export PATH=$PATH:$HOME/dotnet
-   ```
-
-# [.NET Core 1.x](#tab/netcore1x)
+To install .NET Core 1.x for openSUSE (64 bit):
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: step 6 below seems out of place as well

**Review by mairaw — CHANGES_REQUESTED** [review]
I wonder if this topic should be more similar to the other prerequisites topics where we just list the prerequisites and don't list any installation instructions for any .NET Core version. Or do we still need to list 1.x instructions and versions here? /cc @leecow

👍

**Comment by mairaw** [comment]
Also this document is the only prereqs one where people will need to leave the topic to see the specific versions. Only a small percentage of folks visiting the mac and Windows versions need to click that link. Yes, I know the info is duplicated like that, but I think this kind of info shouldn't only be in a repo md file which doesn't get localized.

File: docs/core/linux-prerequisites.md (hunk 5 above)
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: It is. Should have put WIP on it

**Comment by JRAlexander** [comment]
@mairaw - Agreed. I think we can leave the versions at the top. Is that ok with you, @leecow?

**Comment by leecow** [comment]
@JRAlexander - yep, makes sense.

**Review by mairaw — COMMENTED** [review]
A few more comments

File: docs/core/linux-prerequisites.md [hunk 6]
```diff
@@ -24,21 +24,22 @@ This article shows the dependencies needed to develop .NET Core applications on
 
 # [.NET Core 2.x](#tab/netcore2x)
 
-.NET Core 2.0 treats Linux as a single operating system. There is a single Linux build (per chip architecture) for supported Linux distros.
+.NET Core 2.x treats Linux as a single operating system. There is a single Linux build (per chip architecture) for supported Linux distros.
```
[the remark this answers is not in the document]
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: Distros is an accepted term within the Linux community. Would Distributions be better for localization?

[the remark this answers is not in the document]
Comment (mairaw) [reply docs/core/linux-prerequisites.md]: I double checked this term with the style guide team inside C+E. This is what they said:

"The CSG team is going to add a new topic that says not to use distro. It’s jargony and requires an explanation as to what it means. Probably not good for localization. And there’s also nothing wrong with distribution. For example, in that Wikipedia article, distribution is used more than 100 times. In a nutshell, distribution is more clear."

File: docs/core/linux-prerequisites.md (hunk 3 above)
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: Yeah, I'm not sure why the tabs are doing that...

Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: No, by oversight...

File: docs/core/linux-prerequisites.md [hunk 7]
```diff
@@ -49,7 +50,7 @@ See [.NET Core 2.x Supported OS Versions](https://github.com/dotnet/core/blob/ma
 * Oracle Linux 7
 * Fedora 24
 * Debian 8.2 or later versions
-* Ubuntu 14.04, Ubuntu 16.04, Ubuntu 16.10\*
+* Ubuntu 17, 16.10\*, 16.04, 14.04
```
[the remark this answers is not in the document]
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: It should be 17.10. 16.10 has been removed.

File: docs/core/linux-prerequisites.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: ok. Good to know.

File: docs/core/linux-prerequisites.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: Nope.

File: docs/core/linux-prerequisites.md [hunk 8]
```diff
@@ -49,7 +50,7 @@ See [.NET Core 2.x Supported OS Versions](https://github.com/dotnet/core/blob/ma
...
  * Ubuntu 16.10 is supported by the latest patch release of .NET Core 1.1
 * Linux Mint 17
 * openSUSE 42.1 or later versions (.NET Core 1.1)
```
[the remark this answers is not in the document]
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: Yup

File: docs/core/linux-prerequisites.md [hunk 9]
```diff
@@ -110,50 +111,13 @@ The installer bash script is used in automation scenarios and non-admin installa
 > [!IMPORTANT]
```
[the remark this answers is not in the document]
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: That was in there at one point...

[the remark this answers is not in the document]
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: Also, someone put in a bare url just above... Correcting.

File: docs/core/linux-prerequisites.md [hunk 10]
```diff
@@ -20,25 +20,29 @@ This article shows the dependencies needed to develop .NET Core applications on
 * [Command-line with your favorite editor](tutorials/using-with-xplat-cli.md)
 * [Visual Studio Code](https://code.visualstudio.com/)
 
+> [!NOTE]
+> The .NET Core SDK (.dotnet.sdk.x.x.x) package is not required for production servers/environments. Only the .NET Core runtime package (.dotnet.runtime.x.x.x) is needed for apps deployed to production environments. The .NET Core runtime is deployed with Standalone apps as part of the deployment, however, it must be deployed for Shared apps as a seperate deployment.
```
Comment (svick) [inline docs/core/linux-prerequisites.md]: Should ".dotnet.sdk.x.x.x" be "dotnet-sdk-x.x.x" instead? Likewise for runtime.

Comment (svick) [inline docs/core/linux-prerequisites.md]: What are Standalone and Shared apps? Are you talking about [framework-dependent and stand-alone deployments](https://docs.microsoft.com/en-us/dotnet/core/deploying/)? I think different articles should use the same terminology.

File: docs/core/linux-prerequisites.md [hunk 11]
```diff
@@ -20,25 +20,29 @@ This article shows the dependencies needed to develop .NET Core applications on
...
- * Linux Mint 18, Linux Mint 17
- * openSUSE 42.2 or later versions
- * SUSE Enterprise Linux (SLES) 12 SP2 or later versions
+* Red Hat Enterprise Linux 7
+* CentOS 7
+* Oracle Linux 7
+* Fedora 27, 26
+* Debian 9, 8.7 or later versions
+* Ubuntu 17.10, 16.04, 14.04
+* Linux Mint 18,17
```
Comment (svick) [inline docs/core/linux-prerequisites.md]: Nit: missing space after comma.

File: docs/core/linux-prerequisites.md [hunk 12]
```diff
@@ -103,162 +106,84 @@ On Linux, there are two installer package choices:
...
 
----
 4. Run the `dotnet --version` command to prove the installation succeeded.
 
      ```bash
      dotnet --version
      ```
 
 For Red Hat .NET channel access registration help, see [Chapter 1 of the .NET Core 1.1 Getting Started Guide](https://access.redhat.com/documentation/en/net-core/1.1/paged/getting-started-guide/) at Red Hat.
+---
```
Comment (svick) [inline docs/core/linux-prerequisites.md]: This makes the previous line render as heading on GitHub. I think you need an empty line above it if you want `<hr>`.

File: docs/core/linux-prerequisites.md [hunk 13]
```diff
@@ -103,162 +106,84 @@ On Linux, there are two installer package choices:
...
 
-## Install .NET Core for Ubuntu 14.04, Ubuntu 16.04, Ubuntu 16.10 & Linux Mint 17, Linux Mint 18 (64 bit)
-
-1. Remove any **previous preview** versions of .NET Core from your system.
+## Install .NET Core for supported Ubuntu and Linux Mint distributions\versions (64 bit)
```
Comment (svick) [inline docs/core/linux-prerequisites.md]: Nit: I think this should use the normal forward slash, not backslash.

File: docs/core/linux-prerequisites.md [hunk 14]
```diff
@@ -449,54 +304,31 @@ To install .NET Core 2.x on Fedora 26 or Fedora 25, or .NET Core 1.x on Fedora 2
...
 
    ```bash
    dotnet --version
    ```
+---
 
-## Install .NET Core for CentOS 7.1 (64 bit) & Oracle Linux 7.1 (64 bit)
-
-To install .NET Core for CentOS 7.1 (64 bit) & Oracle Linux 7.1 (64 bit):
+## Install .NET Core for supported CentOS and Oracle Linux distributions\versions (64 bit)
```
Comment (svick) [inline docs/core/linux-prerequisites.md]: Nit: forward slash instead of backslash here too (and on the line below).

**Comment by mairaw** [comment]
What's the status of this PR @JRAlexander? Is this ready for a new review or is this WIP?

**Comment by JRAlexander** [comment]
It should be ready for a new review, IIRC.

File: docs/core/linux-prerequisites.md (hunk 13 above)
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: Yes, good catch!

File: docs/core/linux-prerequisites.md (hunk 10 above)
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: Yes, that change had been made earlier. Not sure what happened!

File: docs/core/linux-prerequisites.md (hunk 12 above)
Comment (mairaw) [reply docs/core/linux-prerequisites.md]: this is the syntax for the tabs, but I agree with you that it probably needs an empty line above it @svick. Right now the layout is broken here.

File: docs/core/linux-prerequisites.md [hunk 15]
```diff
@@ -20,25 +20,29 @@ This article shows the dependencies needed to develop .NET Core applications on
...
- * Ubuntu 17.04, Ubuntu 16.04, Ubuntu 14.04
- * Linux Mint 18, Linux Mint 17
- * openSUSE 42.2 or later versions
- * SUSE Enterprise Linux (SLES) 12 SP2 or later versions
+* Red Hat Enterprise Linux 7
+* CentOS 7
+* Oracle Linux 7
+* Fedora 27, 26
+* Debian 9, 8.7 or later versions
+* Ubuntu 17.10, 16.04, 14.04
```
Comment (aaronfranke) [inline docs/core/linux-prerequisites.md]: Ubuntu 18.04 will be released this month. We may as well add it to the list now.

**Comment by aaronfranke** [comment]
This PR has several commits and some of them are minor fixups like formatting.

Consider rebasing the commits. `git rebase -i 39402d3bb7237c9c0f8ce305d359629a21194867` will allow you to rebase from that commit. Change the word "pick" to "fixup" for every minor commit. I'm not sure what the policy is for C# docs, but decent-sized commits help with keeping track of changes.

**Comment by BillWagner** [comment]
@aaronfranke this PR is also affected by the work for #4789  🚧 We're working on it.

File: docs/core/linux-prerequisites.md (hunk 15 above)
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: @aaronfranke, as soon as Ubuntu 18.04 is supported, we'll be sure to add it. Thanks!

**Comment by JRAlexander** [comment]
Closing and reopening after build failure to retrigger build.

**Comment by JRAlexander** [comment]
Closing and reopening

File: docs/core/linux-prerequisites.md [hunk 16]
```diff
@@ -103,537 +105,288 @@ On Linux, there are two installer package choices:
 
 ### Scripting Installs with the .NET Core installer script
 
-The `dotnet-install` scripts are used to perform a non-admin install of the CLI toolchain and the shared runtime. You can download the script from: https://dot.net/v1/dotnet-install.sh
+The [`dotnet-install` scripts](./tools/dotnet-install-script.md) are used to perform a non-admin install of the CLI toolchain and the shared runtime. You can download the script from [https://dot.net/v1/dotnet-install.sh](https://dot.net/v1/dotnet-install.sh).
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: might be better to remove the inline code from `dotnet-install`. It looks a bit weird in the middle of a link:
image

File: docs/core/linux-prerequisites.md [hunk 17]
```diff
@@ -103,537 +105,288 @@ On Linux, there are two installer package choices:
...
 
 The installer bash script is used in automation scenarios and non-admin installations. This script also reads PowerShell switches, so they can be used with the script on Linux/OS X systems.
 
 > [!IMPORTANT]
 > Before running the script, install the required [dependencies](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md).
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: this just links back to this topic. maybe link to the #linux-distribution-dependencies section?

File: docs/core/linux-prerequisites.md [hunk 18]
```diff
@@ -103,537 +105,288 @@ On Linux, there are two installer package choices:
...
-   ```
+1. Remove any **previous preview** versions of .NET Core from your system.
 
-6. Run the `dotnet --version` command to prove the installation succeeded.
+2. Install .NET Core 1.x on Debian 9 or Debian 8:
 
-   ```bash
-   dotnet --version
-   ```
+* [.NET Core Runtime 1.1.7 install link](https://www.microsoft.com/net/download/thank-you/dotnet-runtime-1.1.7-linux-debian-x64-binaries)
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: should this follow the same pattern as the other sections and be on a table (even though there is only one column)?

File: docs/core/linux-prerequisites.md [hunk 19]
```diff
@@ -103,537 +105,288 @@ On Linux, there are two installer package choices:
...
 
-4. Run the `dotnet --version` command to prove the installation succeeded.
+2. Install .NET Core 1.x on supported Ubuntu and Linux Mint distributions/versions (64 bit):
 
-   ```bash
-   dotnet --version
-   ```
+| Runtimes / SDKs         |Ubuntu 16.04 / Linux Mint 18|Ubuntu 14.04 / Linux Mint 17|
+|-------------------------|----------------------------|----------------------------|
+|.NET Core Runtime 1.1.7  |[Install link](https://www.microsoft.com/net/download/thank-you/dotnet-runtime-1.1.7-linux-ubuntu-16.04-x64-binaries)            |[Install link](https://www.microsoft.com/net/download/thank-you/dotnet-runtime-1.1.7-linux-ubuntu-14.04-x64-binaries)            |
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: @rowanmiller is there any query params that we can use in these links for you to track that these downloads are coming from docs?

File: docs/core/linux-prerequisites.md [hunk 20]
```diff
@@ -103,537 +105,288 @@ On Linux, there are two installer package choices:
...
+|.NET Core SDK 2.0.3      |[Install link](https://www.microsoft.com/net/download/linux-package-manager/debian9/sdk-2.0.3)   |[Install link](https://www.microsoft.com/net/download/linux-package-manager/debian8/sdk-2.0.3)   |
 
-   ```bash
-   curl -sSL -o dotnet.tar.gz https://go.microsoft.com/fwlink/?linkid=848826
-   ```
+**.NET Core 2.1**
 
-4. Extract the .NET Core SDK binaries.
+>[!IMPORTANT]
+> To use .NET Core 2.1 with Visual Studio, you'll need to [install Visual Studio 2017 15.7 Preview 1 or newer](https://www.visualstudio.com/vs/preview/?dotnetid=169771757.1498243949).
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: remove future tense and do you need that dotnetid there? should you mention when to expect support for VS for Mac (see https://blogs.msdn.microsoft.com/dotnet/2018/04/11/announcing-net-core-2-1-preview-2/)?

@TerryGLee is there any special campaign link for the preview version?

**Review by mairaw — COMMENTED** [review]
Looks great! Left a couple of comments...

File: docs/core/linux-prerequisites.md (hunk 18 above)
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: I went back and forth with this. It fills the entire page with both columns on the sides and looks really strange.

File: docs/core/linux-prerequisites.md (hunk 19 above)
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: Great question!

Comment (rowanmiller) [reply docs/core/linux-prerequisites.md]: Not required for links like this. We get referral information by default, so we know which traffic is coming from docs (and what page it comes from).

File: docs/core/linux-prerequisites.md [hunk 21]
```diff
@@ -103,537 +105,288 @@ On Linux, there are two installer package choices:
...
-   ```
+To install .NET Core on supported RHEL versions:
 
-Enable .NET Core 2.0 SDK/Runtime for your environment:
+# [.NET Core 2.x](#tab/netcore2x)
 
-   ```bash
-   scl enable rh-dotnet20 bash
-   ```
+To ensure you have the latest installation information, follow the [.NET Core 2.x SDK and Runtime Installer instructions](https://www.microsoft.com/net/learn/get-started/linuxredhat) for supported RHEL versions.
```
Comment (rowanmiller) [inline docs/core/linux-prerequisites.md]: I think this should probably point to just the install instructions right, and not the Get Started tutorial? https://www.microsoft.com/net/download/linux-package-manager/rhel/sdk-current

Comment (rowanmiller) [reply docs/core/linux-prerequisites.md]: I know for most content on this page you want a specific version, but if you have places where you just want current, then you can replace the version number in the URL with `current` (like the above link)

File: docs/core/linux-prerequisites.md [hunk 22]
```diff
@@ -20,25 +20,28 @@ This article shows the dependencies needed to develop .NET Core applications on
 * [Command-line with your favorite editor](tutorials/using-with-xplat-cli.md)
 * [Visual Studio Code](https://code.visualstudio.com/)
 
+> [!NOTE]
+> The .NET Core SDK (.dotnet-sdk-x-x64.x.x.x) package is not required for production servers/environments. Only the .NET Core runtime package (.dotnet-x-x64.x.x.x) is needed for apps deployed to production environments. The .NET Core runtime is deployed with apps as part of a self-contained deployment, however, it must be deployed for Framework-dependent deployed apps seperately. For general information about Framework-dependent and Self-contained deployment types, please see [.NET Core application deployment](https://docs.microsoft.com/en-us/dotnet/core/deploying/). Also see [Self-contained Linux applications](https://github.com/dotnet/core/blob/master/Documentation/self-contained-linux-apps.md) for specific guidelines.
```
[the remark this answers is not in the document]
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: I removed the filenames. Yes, it's a package.

[the remark this answers is not in the document]
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: I put a note on my board to port over the other doc. :)

**Comment by JRAlexander** [comment]
closing and reopening

File: docs/core/linux-prerequisites.md [hunk 23]
```diff
@@ -20,25 +20,28 @@ This article shows the dependencies needed to develop .NET Core applications on
 * [Command-line with your favorite editor](tutorials/using-with-xplat-cli.md)
 * [Visual Studio Code](https://code.visualstudio.com/)
 
+> [!NOTE]
+> The .NET Core SDK package is not required for production servers/environments. Only the .NET Core runtime package is needed for apps deployed to production environments. The .NET Core runtime is deployed with apps as part of a self-contained deployment, however, it must be deployed for Framework-dependent deployed apps seperately. For more information about framework-dependent and self-contained deployment types, see [.NET Core application deployment](https://docs.microsoft.com/en-us/dotnet/core/deploying/). Also see [Self-contained Linux applications](https://github.com/dotnet/core/blob/master/Documentation/self-contained-linux-apps.md) for specific guidelines.
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: relative link to the deployment topic: `[./deploying/index.md]`

typo: seperately

File: docs/core/linux-prerequisites.md [hunk 24]
```diff
@@ -103,537 +105,279 @@ On Linux, there are two installer package choices:
...
+* .NET Core Runtime 2.0.6 [install link](https://www.microsoft.com/net/download/linux-package-manager/centos/runtime-2.0.6)
+* .NET Core Runtime 2.0.5 [install link](https://www.microsoft.com/net/download/linux-package-manager/centos/runtime-2.0.5)
+* .NET Core SDK 2.1.103 [install link](https://www.microsoft.com/net/download/linux-package-manager/centos/sdk-2.1.103)
+* .NET Core SDK 2.0.3 [install link](https://www.microsoft.com/net/download/linux-package-manager/centos/sdk-2.0.3)
+ 
+**.NET Core 2.1**
 
-5. Add dotnet to your PATH
+>[!IMPORTANT]
+> To use .NET Core 2.1 with Visual Studio, you need to [install Visual Studio 2017 15.7 Preview 1 or newer](https://www.visualstudio.com/vs/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=banner+cta&utm_content=vs2017).
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: in this case, the non-campaign link was better: https://www.visualstudio.com/vs/preview/

File: docs/core/linux-prerequisites.md [hunk 25]
```diff
@@ -103,537 +105,279 @@ On Linux, there are two installer package choices:
...
-   ```
-
 > [!IMPORTANT]
-> If you have problems with the .NET Core 2.x installation on a supported Linux distribution/version, consult the [2.0 Known issues](https://github.com/dotnet/core/tree/master/release-notes/2.0) topic for your installed distributions/versions. 
+> If you have problems with the .NET Core 2.x installation on a supported Linux distribution/version, consult the following topics for your installed distributions/versions:
+> * [.NET Core 2.0 known issues](https://github.com/dotnet/core/tree/master/release-notes/2.0)
+> * [.NET Core 2.1 known issues](https://github.com/dotnet/core/tree/master/release-notes/2.1)
 >
-> If you have problems with the .NET Core 1.x installation on a supported Linux distribution/version, consult the [1.0.0 Known Issues](https://github.com/dotnet/core/blob/master/release-notes/1.0/1.0.0-known-issues.md) and [1.0.1 Known Issues](https://github.com/dotnet/core/blob/master/release-notes/1.0/1.0.1-known-issues.md) topics for your installed distributions/versions.
+> If you have problems with the .NET Core 1.x installation on a supported Linux distribution/version, consult the following topics for your installed distributions/versions:
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: nit: not sure if you need to say this again. Maybe a list with all known issues in sequence?

**Review by mairaw — APPROVED** [review]
This looks great @JRAlexander. I'm approving it now since it's close to being done but I still found some minor issues to be fixed. Thanks!

File: docs/core/linux-prerequisites.md (hunk 23 above)
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: Good catch!

**Comment by JRAlexander** [comment]
@mairaw, as always, very thoughtful and great feeedback! Thank you!!!
